index.html

```diff
@@ -44,7 +45,6 @@ <h3>Here's a time line of Dr. Borlaug's life:</h3>
             <p>If you have time, you should read more about this incredible human being on his <a id ="tribute-link" href="#" target="_blank">Wikipedia entry</a>.</p>
             
         </div>
-
     </div>
```

Comment:
- [ ]  Note that you leave an unnecessary indentation at the end of your page. Please remove it and change the structure.

Reply:
Well done